src/cnvgrib/putgbexn.f

```diff
@@ -1,170 +1,146 @@
+!> - 3 grid definition
+!> - 4 gds/bms flag (right adj copy of octet 8)
+!> - 5 indicator of parameter
+!> - 6 type of level
+!> - 7 height/pressure , etc of level
+!> - 8 year including (century-1)
+!> - 9 month of year
+!> - 10 day of month
+!> - 11 hour of day
+!> - 12 minute of hour
+!> - 13 indicator of forecast time unit
+!> - 14 time range 1
+!> - 15 time range 2
+!> - 16 time range flag
+!> - 17 number included in average
+!> - 18 version nr of grib specification
+!> - 19 version nr of parameter table
+!> - 20 nr missing from average/accumulation
+!> - 21 century of reference time of data
+!> - 22 units decimal scale factor
+!> - 23 subcenter number
+!> - 24 pds byte 29, for nmc ensemble products, 128 if forecast field
+!> error, 64 if bias corrected fcst field, 32 if smoothed field,
+!> warning: can be combination of more than 1.
+!> - 25 pds byte 30, not used
+!> @param[in] kgds teger (200) gds parameters
+!> - 1 data representation type
+!> - 19 number of vertical coordinate parameters
+!> - 20 octet number of the list of vertical coordinate parameters or
+!> octet number of the list of numbers of points in each row or 255 if
+!> neither are present.
+!> - 21 for grids with pl, number of points in grid
+!> - 22 number of words in each row latitude/longitude grids
+!> - 2 n(i) nr points on latitude circle
+!> - 3 n(j) nr points on longitude meridian
+!> - 4 la(1) latitude of origin
+!> - 5 lo(1) longitude of origin
+!> - 6 resolution flag (right adj copy of octet 17)
+!> - 7 la(2) latitude of extreme point
+!> - 8 lo(2) longitude of extreme point
+!> - 9 di longitudinal direction of increment
+!> - 10 dj latitudinal direction increment
+!> - 11 scanning mode flag (right adj copy of octet 28)
+!> Gaussian  grids:
+!> - 2 n(i) nr points on latitude circle
+!> - 3 n(j) nr points on longitude meridian
+!> - 4 la(1) latitude of origin
+!> - 5 lo(1) longitude of origin
+!> - 6 resolution flag  (right adj copy of octet 17)
+!> - 7 la(2) latitude of extreme point
+!> - 8 lo(2) longitude of extreme point
+!> - 9 di longitudinal direction of increment
+!> - 10 n - nr of circles pole to equator
+!> - 11 scanning mode flag (right adj copy of octet 28)
+!> - 12 nv - nr of vert coord parameters
+!> - 13 pv - octet nr of list of vert coord parameters or pl - location
+!> of the list of numbers of points in each row (if no vert coord
+!> parameters are present or 255 if neither are present
+!> Polar Stereographic grids:
+!> - 2 n(i) nr points along lat circle
+!> - 3 n(j) nr points along lon circle
+!> - 4 la(1) latitude of origin
+!> - 5 lo(1) longitude of origin
+!> - 6 resolution flag  (right adj copy of octet 17)
+!> - 7 lov grid orientation
+!> - 8 dx - x direction increment
+!> - 9 dy - y direction increment
+!> - 10 projection center flag
+!> - 11 scanning mode (right adj copy of octet 28)
+!> Spherical Harmonic Coefficients:
+!> - 2 j pentagonal resolution parameter
+!> - 3 k      "          "         "
+!> - 4 m      "          "         "
```

Comment:
To keep consistent I have been writing out 'pentagonal resolution parameter' where the quotes are.

Reply:
Fixed.